datafusion/core/tests/sql/explain_analyze.rs

```diff
@@ -136,8 +137,16 @@ async fn explain_analyze_baseline_metrics() {
             }
             let metrics = plan.metrics().unwrap().aggregate_by_partition();
 
-            assert!(metrics.output_rows().unwrap() > 0);
-            assert!(metrics.elapsed_compute().unwrap() > 0);
+            assert!(
+                metrics.output_rows().unwrap() > 0,
+                "plan: {}",
+                DisplayableExecutionPlan::with_metrics(plan).one_line()
+            );
+            assert!(
```

Comment:
This change appears to be intermittently failing

https://github.com/apache/arrow-datafusion/runs/6100177086?check_suite_focus=true

https://github.com/apache/arrow-datafusion/runs/6100277301?check_suite_focus=true

https://github.com/apache/arrow-datafusion/runs/6100969854?check_suite_focus=true